masterlist.yaml

```diff
@@ -1839,6 +1839,13 @@ globals:
       - '[.NET Script Framework](https://www.nexusmods.com/skyrimspecialedition/mods/21294/)'
     condition: 'file("DLLPlugins/NetScriptFramework.Runtime.dll") and ((not file("../binkw64_.dll") and not checksum("../binkw64.dll", 6C2F54AB)) and (not file("../d3dx9_42.dll")))'
 
+# Other Missing Requirements
+  - <<: *missingRequirementXforY
+    subs:
+      - '[Unique Region Names SE](https://www.nexusmods.com/skyrimspecialedition/mods/56452/)'
+      - '[Regional Save Names](https://www.nexusmods.com/skyrimspecialedition/mods/49698/)'
+    condition: 'file("SKSE/Plugins/po3_RegionalSaveNames.dll") and not active("Unique Region Names.esp")'
+
```

Comment:
Globals are usually for messages where there no plugin provided with the mod.

```suggestion
```

---

If you don't want to use the `req:` key, then you can use a `*requiresX` message attached to `Unique Region Names.esp`.

```yaml
  - name: 'Unique Region Names.esp'
    msg:
      - <<: *requiresX
        subs: [ '[Regional Save Names](https://www.nexusmods.com/skyrimspecialedition/mods/49698/)' ]
        condition: 'file("SKSE/Plugins/po3_RegionalSaveNames.dll")'
```

Reply:
This works perfectly, I had wrongly assumed this would only be evaluated when the listed plugin was present and `globals` was the use case for this. If I used the `req` key, wouldn't it make Unique Region Names depend on Regional Save Names?